test/dleq.test.ts

```diff
@@ -7,20 +7,19 @@ import {
     type DLEQParams,
     type Elt,
     type Scalar,
+    CryptoImpl,
     DLEQProof,
-    DLEQProver,
-    Oprf
+    DLEQProver
 } from '../src/index.js'
-import { describeGroupTests } from './describeGroupTests.js'
+import { describeCryptoTests } from './describeCryptoTests.js'
 import { serdeClass } from './util.js'
 
-describeGroupTests((g) => {
+describeCryptoTests((g) => {
     describe.each(g.supportedGroups)('DLEQ', (id) => {
         const groupName = id
-        const gg = Oprf.Crypto.Group.fromID(id)
+        const gg = CryptoImpl.Group.fromID(id)
```

Comment:
the variable `g` is the group, so no need to define `gg` ?

Reply:
Yeah, I guess not :)

Reply:
Oh, no, g is the `GroupCons`, so `Group` 

And I guess it makes sense to try and use (and thus test?) the standard way of accessing it - via CryptoImpl.Group

Reply:
Updated the helper to only pass the supportedGroups/supportedSuites